openpifpaf/headmeta.py

```diff
@@ -48,6 +48,8 @@ class Cif(Base):
     decoder_min_scale = 0.0
     decoder_seed_mask: List[int] = None
 
+    weights: List[float] = None
```

Comment:
Sorry, one more request (naming is important), please change this to `training_weights`.